Packs/AWS-EC2/Integrations/AWS-EC2/AWS-EC2.py

```diff
@@ -1548,6 +1548,74 @@ def authorize_security_group_ingress_command(args, aws_client):
         demisto.results("The Security Group ingress rule was created")
 
 
+def authorize_security_group_egress_command(args, aws_client):
+    client = aws_client.aws_session(
+        service='ec2',
+        region=args.get('region'),
+        role_arn=args.get('roleArn'),
+        role_session_name=args.get('roleSessionName'),
+        role_session_duration=args.get('roleSessionDuration'),
+    )
+    kwargs = {'GroupId': args.get('groupId')}
+    IpPermissions = []
+    IpPermissions_dict = {}
+    UserIdGroupPairs = []
+    UserIdGroupPairs_dict = {}
+
+    if args.get('IpPermissionsfromPort') is not None:
+        IpPermissions_dict.update({'FromPort': int(args.get('IpPermissionsfromPort'))})
+    if args.get('IpPermissionsIpProtocol') is not None:
+        IpPermissions_dict.update({'IpProtocol': str(args.get('IpPermissionsIpProtocol'))})  # type: ignore
+    if args.get('IpPermissionsToPort') is not None:
+        IpPermissions_dict.update({'ToPort': int(args.get('IpPermissionsToPort'))})
+
+    if args.get('IpRangesCidrIp') is not None:
+        IpRanges = [{
+            'CidrIp': args.get('IpRangesCidrIp'),
+            'Description': args.get('IpRangesDesc', None)
+        }]
+        IpPermissions_dict.update({'IpRanges': IpRanges})  # type: ignore
+    if args.get('Ipv6RangesCidrIp') is not None:
+        Ipv6Ranges = [{
+            'CidrIp': args.get('Ipv6RangesCidrIp'),
+            'Description': args.get('Ipv6RangesDesc', None)
+        }]
+        IpPermissions_dict.update({'Ipv6Ranges': Ipv6Ranges})  # type: ignore
+    if args.get('PrefixListId') is not None:
+        PrefixListIds = [{
+            'PrefixListId': args.get('PrefixListId'),
+            'Description': args.get('PrefixListIdDesc', None)
+        }]
+        IpPermissions_dict.update({'PrefixListIds': PrefixListIds})  # type: ignore
+
+    if args.get('UserIdGroupPairsDescription') is not None:
```

Comment:
this huge chunk of code is used 3 times in this integration. Please refactor so all commands use 1 function.